mcs/packages/Makefile

```diff
@@ -3,7 +3,7 @@ include ../build/rules.make
 
 ROSLYN_CSC_DIR = $(dir $(CSC_LOCATION))
 
-ROSLYN_FILES = \
+ROSLYN_FILES_FOR_MONO = \
```

Comment:
The duplicated list is unfortunate but symlinking the whole directory to `lib/mono/4.5` sounds a bit dangerous (who knows how that influences the assembly reference probing/resolving that msbuild does). I'd lean on keeping the separate folder for now.